### PR TITLE
fix ceilometer serverspec errors

### PR DESCRIPTION
**Pull request by davidcusatis**

### CODE REVIEW

File: roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb [hunk 1]
```diff
@@ -20,8 +20,16 @@
...
+processes = ['ceilometer-api',
+  'ceilometer-collector',
+  'ceilometer-agent-notification',
+  'ceilometer-agent-central',
+  'ceilometer-alarm-evaluator',
+  'ceilometer-alarm-notifier']
+processes.each do |process|
+  describe process(process) do
+    its(:user) { should eq 'ceilometer' }
+  end
```
Comment (j2sol) [inline roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb]: Do these processes exist everywhere that Ceilometer role is ran on?

**Comment by j2sol** [comment]
ok to test

File: roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb (hunk 1 above)
Comment (davidcusatis) [inline roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb]: to be completely honest, I don't know enough about ceilometer to give you a 100% answer. But I did read the ceilometer architecture overview (http://docs.openstack.org/developer/ceilometer/architecture.html) and it seems all of these processes are needed for ceilometer to work.

**Comment by epwn** [comment]
Verified on my environment :+1:

**Pull request merged**